pkg/parser/endpoints.wbnf

```diff
@@ -17,6 +17,13 @@ field_type -> collection_type | (type_spec optional="?"? attribs?) ;
 
 collector -> '.. * <- *' ":" (SHORTCUT | %!Indented(collector_stmts));
 
-collector_stmts ->  (
-                        call=(target=APPLICATION_NAME "<-" target_endpoint=NAME) | action=APPLICATION_NAME
-                    ) attribs;
+collector_stmts -> collector_http_stmt attribs
+                    | collector_pubsub_call attribs
+                    | collector_call_stmt attribs
+                    | action=APPLICATION_NAME attribs;
+
+collector_call_stmt -> target=APPLICATION_NAME "<-" target_endpoint=APPLICATION_NAME;
```

Comment:
I don't think endpoints should support `::`. This is actually a good reason to split the bottom of the `APPLICATION_NAME` stack into a separate rule.